app/lib/actions/katello/host/hypervisors.rb

```diff
@@ -0,0 +1,22 @@
+module Actions
+  module Katello
+    module Host
+      class Hypervisors < Actions::EntryAction
+        def plan(environment, content_view, hypervisor_params)
+          sequence do
+            hypervisor_results = plan_action(::Actions::Candlepin::Consumer::Hypervisors, hypervisor_params)
+            return if hypervisor_results.error
```

Comment:
I based this off of register.rb[1]; is it incorrect there as well?

[1] https://github.com/Katello/katello/blob/host_unification/app/lib/actions/katello/host/register.rb#L33